Plugins/Public/base_plugin/CoreModule.cpp

```diff
@@ -132,6 +132,8 @@ void CoreModule::Spawn()
 		
 		pub::SpaceObj::SetRelativeHealth(space_obj, base->base_health / base->max_base_health);
 
+		base->baseCSolar = (CSolar*)CObject::Find(space_obj, CObject::CSOLAR_OBJECT);
```

Comment:
You must call Release on this whenever we bin a PlayerBase object, e.g. probably in basedespawn